mkdocs.yml

```diff
@@ -45,67 +45,101 @@ plugins:
-        - 'Operation': 'dev/components/operation.md'
-        - 'Serialisation': 'dev/components/serialisation.md'
-        - 'Integration Tests': 'dev/components/integration-test.md'
-        - 'Core REST': 'dev/components/core-rest.md'
-        - 'Spring REST': 'dev/components/spring-rest.md'
-        - 'Accumulo Store': 'dev/components/accumulo-store.md'
-        - Libraries:
-          - dev/components/libraries/bitmap.md
-          - dev/components/libraries/flink.md
-          - dev/components/libraries/sketches.md
-          - dev/components/libraries/spark.md
-          - dev/components/libraries/time.md
-    - 'Reference':
+        - 'Project Setup': 'development-guide/example-deployment/project-setup.md' 
+        - 'Writing the Schema': 'development-guide/example-deployment/writing-the-schema.md'
+        - 'Running the Deployment': 'development-guide/example-deployment/running-the-deployment.md'
+        - 'Using the API': 'development-guide/example-deployment/using-the-api.md'
+      - 'Ways of Working': 'development-guide/ways-of-working.md'
+      - 'Remote Coding Environments': 'development-guide/remote-coding-environments.md'
+      - 'Extending Gaffer': 'development-guide/extending-gaffer.md'
+      - 'Sketches Custom Deserialisation': 'development-guide/rest-api-sketches.md'
+      - Project Structure:
+        - Components:
+          - 'Components/Maven Modules': 'development-guide/project-structure/components/components.md'
+          - 'Cache': 'development-guide/project-structure/components/cache.md'
+          - 'Data': 'development-guide/project-structure/components/data.md'
+          - 'Graph': 'development-guide/project-structure/components/graph.md'
+          - 'Store': 'development-guide/project-structure/components/store.md'
+          - 'Operation': 'development-guide/project-structure/components/operation.md'
+          - 'Serialisation': 'development-guide/project-structure/components/serialisation.md'
+          - 'Integration Tests': 'development-guide/project-structure/components/integration-test.md'
+          - 'Core REST': 'development-guide/project-structure/components/core-rest.md'
+          - 'Spring REST': 'development-guide/project-structure/components/spring-rest.md'
+          - 'Accumulo Store': 'development-guide/project-structure/components/accumulo-store.md'
+          - Libraries:
+            - development-guide/project-structure/components/libraries/bitmap.md
+            - development-guide/project-structure/components/libraries/flink.md
+            - development-guide/project-structure/components/libraries/sketches.md
+            - development-guide/project-structure/components/libraries/spark.md
+            - development-guide/project-structure/components/libraries/time.md
+        - Maven Dependencies:
+          - 'Spring': 'development-guide/project-structure/maven-dependencies/spring.md'
+    - Administration Guide:
+      - 'Getting Started':
```

Comment:
Think the admin guide probably needs the existing quickstart page in just to show how to get up and running as fast as possible